lib/amplitude_web.dart

```diff
@@ -27,6 +29,30 @@ class AmplitudeFlutterPlugin {
   /// Note: Check the "federated" architecture for a new way of doing this:
   /// https://flutter.dev/go/federated-plugins
   Future<dynamic> handleMethodCall(MethodCall call) async {
+    if (call.method == 'init') {
+      var args = call.arguments;
+      String apiKey = args['apiKey'];
+      JSObject configuration = getConfiguration(call);
+
+      // Set library
+      Amplitude instance = amplitude.createInstance();
+      instance.add(createJSInteropWrapper(FlutterLibraryPlugin(
+          args['library'] ?? 'amplitude_flutter/unknown')));
+      instance.init(apiKey, configuration);
+
+      instances[args['instanceName'] ?? '\$default_instance'] = instance;
+
+      return null;
+    }
+
+    Amplitude? instance =
+        instances[call.arguments['instanceName'] ?? '\$default_instance'];
+
+    if (instance == null) {
+      throw Exception(
+          'instance not found: ${call.arguments['instanceName'] ?? '\$default_instance'}');
+    }
+
     switch (call.method) {
       case "init":
```

Comment:
Should we just remove the `init` block from this switch since it's handled by the `if (call.method == 'init'){` logic you added on L32 that returns early?

Reply:
Sure! I forgot to clean that up